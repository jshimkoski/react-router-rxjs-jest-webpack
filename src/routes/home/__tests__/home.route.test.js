import React from 'react';
import Home from '../home.route';
import renderer from 'react-test-renderer';

describe('Home', () => {
  let component;
  beforeEach(() => {
    component = renderer.create(
      <Home CounterState={{counter: 1}} />
    ).toJSON();
  });

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});