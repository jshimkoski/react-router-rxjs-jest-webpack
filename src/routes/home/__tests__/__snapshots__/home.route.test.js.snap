exports[`Home renders correctly 1`] = `
<section
  className="home">
  <h2>
    Home
  </h2>
  <button
    onClick={[Function]}>
    Increase
  </button>
  <button
    onClick={[Function]}>
    Decrease
  </button>
  1
</section>
`;
